Sources/Flows/OAuth2PasswordGrant.swift

```diff
@@ -19,14 +19,30 @@
 //
 
 import Foundation
+
 #if !NO_MODULE_IMPORT
+
 import Base
+
 #endif
 
+/**
+An object adopting this protocol is responsible of the creation of the login controller
+*/
+
+public protocol OAuth2PasswordGrantDelegate: class {
+	/**
+	Instanciates and configures the login controller to present.
+	Don't forget setting it's oauth2 instance with the one in parameter.
```

Comment:
Add an empty line between the one-sentence explanation and extra text. Helps when generating docs. Also should be "its", not "it's".